controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -430,7 +430,7 @@ public String getScopeName() {
         } else {
             log.debug("scale conflict for stream {}/{} with segments to seal {}", scope, name, sealedSegments);
 
-            return FutureHelpers.failedFuture(new ScaleOperationExceptions.ScaleStartException());
```

Comment:
This is the only place where this exception is used in the code base other than tests. Should we remove `ScaleStartException` altogether?

Reply:
I am unifying all this in PR for issue 1738 that i am working on currently. 
We will need to do something similar for all stream requests where for any manual request : 
1. we post event for work
2. start work
3. inform caller about work being started.

Now we could have picked event right after (1) for processing before (2) could be performed.
In such cases we want to throw StartException and retry a few times to give the caller a chance to initiate the work.